sample-code/css-collapsed-sections.html

```diff
@@ -0,0 +1,747 @@
+  font-stretch: 100%;
+  line-height: 1.65;
+}
+
+div,body {
+  margin: 0;
+  padding: 0;
+  border: 0;
+}
+#page {
+  margin: 0 16px;
+}
+.heading {
+  margin-bottom: 12px;
+  font-size: 16px;
+  border-bottom: 1px solid #eaecf0;
+}
+.main-heading {
+  border-bottom: 1px solid #c8ccd1;
+}
+h1 {
+  line-height: 1.3;
+  font-family: 'Linux Libertine', 'Georgia', 'Times', serif;
+  font-size: 1.7em;
+  font-weight: 400;
+}
+h2 {
+  display: table;
+  vertical-align: baseline;
+  margin: 0;
+  border-bottom: 1px solid #eaecf0;
+  margin-bottom: 0.5em;
+  position: relative;
+  clear: left;
+  cursor: pointer;
+  padding: 0.5em 0;
+  line-height: 1.3;
+  font-family: 'Linux Libertine', 'Georgia', 'Times', serif;
+  font-size: 1.5em;
+  font-weight: 400;
+  width: 100%;
+}
+h3 {
+  margin: 0;
+  padding: 0.5em 0;
+}
+.note {
+  padding: 5px 7px;
+  color: #54595d;
+  font-size: 0.8125em;
+  background-color: #f8f9fa;
+  margin-bottom: 1px;
+  overflow: hidden;
+}
+table {
+  font-size: 90%;
+  position: relative;
+  border: 1px solid #eaecf0;
+  margin-bottom: 2em;
+  background-color: #f8f9fa;
+  display: flex;
+  flex: 1 1 100%;
+  flex-flow: column nowrap;
+  margin-left: 0;
+  margin-right: 0;
+  border-collapse: collapse;
+}
+caption {
+  display: flex;
+  flex-flow: column nowrap;
+  padding: 10px 10px 0;
+  text-align: center;
+  font-weight: bold;
+}
+tr {
+  min-width: 100%;
+  display: flex;
+  flex-flow: row nowrap;
+  border-spacing: 2px;
+}
+th {
+  flex: 1 0;
+  padding: 7px 10px;
+  border: 0;
+  border-bottom: 1px solid #eaecf0;
+}
+td {
+  flex: 1 0;
+  padding: 7px 10px;
+  border: 0;
+  border-bottom: 1px solid #eaecf0;
+}
+sup {
+  font-size: 0.75em;
+  line-height: 1;
+  vertical-align: super;
+}
+.arrow {
+  margin-left: -0.5625em;
+  padding: 0.5625em;
+  width: 1em;
+  height: 1em;
+  line-height: 1em;
+  border-radius: 2px;
+  color: transparent;
+  font-size: initial;
+  position: relative;
+  display: inline-block;
+  box-sizing: content-box !important;
+  background-color: transparent;
+}
+.arrow.up::before {
+  transform: scaleY(-1);
+}
+.arrow.down::before {
+  transform: scaleY(1);
+}
+.arrow::before {
+  background-image: linear-gradient(transparent,transparent),url("data:image/svg+xml,%3Csvg xmlns=%22http://www.w3.org/2000/svg%22 width=%2220%22 height=%2220%22 viewBox=%220 0 20 20%22%3E %3Ctitle%3E expand %3C/title%3E %3Cpath d=%22M17.5 4.75l-7.5 7.5-7.5-7.5L1 6.25l9 9 9-9z%22/%3E %3C/svg%3E");
+  content: '';
+  display: block;
+  width: 100%;
+  height: 100%;
+  min-width: 1em;
+  min-height: 1em;
+  background-size: 1em 1em;
+  background-position: center;
+}
+span {
+  width: 100%;
+  display: table-cell;
+  vertical-align: middle;
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  cursor: pointer;
+}
+</style>
+
+<!-- most of the page is just hardcoded chunks of Wikipedia's Wikipedia page
+     comment below are on render-subtree specific bits
+-->
+<div id=page>
```

Comment:
Put some text at the top in large font saying that this is a demo of direct integration with Wikipedia

Reply:
Done.